docs/Installation.md

```diff
@@ -241,9 +174,9 @@ sudo systemctl start zm-testagent
 See the [post-installation] section for post-installation matters.
 
 
-## 4. Installation on Debian
+## 4. Installation on Debian and Ubuntu
 
-### 4.1 Install Zonemaster::Backend and related dependencies (Debian)
+### 4.1 Install Zonemaster::Backend and related dependencies (Debian and Ubuntu)

```

Comment:
```
### 4.1 Install Zonemaster::Backend and related dependencies (Debian/Ubuntu)
```

Reply:
done, please note that doing so produces the following fragment `#41-install-zonemasterbackend-and-related-dependencies-debianubuntu` (Debian/Ubuntu -> `debianubuntu`).

Reply:
OK, I do not see that as a problem. An alternative is to create HTML fragments.

Reply:
I don't think we need to go that far. This is fine for me as well.